docs/source/cuda-reference/kernel.rst

```diff
@@ -499,6 +499,13 @@ These functions return a 16-bit floating point result.
 
    Returns the 16-bit floating point result of the multiplication.
 
+.. function:: numba.cuda.fp16.hdiv (a, b)
+
+   Perform the divide operation ``a / b`` on 16-bit floating point arguments in
+   round to nearest mode.
```

Comment:
Should this also note that it is implemented as multiply-by-reciprocal?

Reply:
I personally think this is an implementation detail that it is not necessary to specify.